integration-test/src/test/java/com/datastrato/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -118,7 +117,6 @@ public void startTrinoContainer(
             .withExposePorts(ImmutableSet.of(TrinoContainer.TRINO_PORT))
             .withTrinoConfDir(trinoConfDir)
             .withMetalakeName(metalakeName)
-            .withGravitinoServerAddress(gravitinoServerAddr)
```

Comment:
If we on long need this `withGravitinoServerAddress` function, Please delete this function in the TrionContrianer.